tests/cmd/rm/invalid_rm_target.toml

```diff
@@ -0,0 +1,15 @@
+bin.name = "cargo-rm"
+args = ["rm", "--target", "powerpc-unknown-linux-gnu", "dbus"]
+status.code = 1
+stdout = ""
+stderr = """
+    Removing dbus from dependencies for target `powerpc-unknown-linux-gnu`
+Could not edit `Cargo.toml`.
+
+ERROR: The table `powerpc-unknown-linux-gnu` could not be found.
+Error: The table `powerpc-unknown-linux-gnu` could not be found.
```

Comment:
This is an existing issue, affecting both cargo add and rm. In both cases, it's caused by a duplicate `eprintln`, one of which is a side effect in a `map_err`. For example, in rm:
https://github.com/killercup/cargo-edit/blob/bab71dbe194d2924287b6905efd6349c39c5e26a/src/bin/rm/rm.rs#L108-L111 https://github.com/killercup/cargo-edit/blob/bab71dbe194d2924287b6905efd6349c39c5e26a/src/bin/rm/main.rs#L24-L25

I didn't fix it this time around since the other tests seemed to expect that duplication. If you'd like, I can open up another issue/PR to fix this!

Reply:
Eh, ignore it.  It'll not exist when we switch to `cargo`.